ginkgo/main.go

```diff
@@ -246,10 +246,15 @@ func findSuites(args []string, recurse bool, skipPackage string, allowPrecompile
 					continue
 				}
 			}
-			suites = append(suites, testsuite.SuitesInDir(arg, recurse)...)
+			recurseForSuite := recurseForAll
+			if strings.HasSuffix(arg, "/...") && arg != "/..." {
```

Comment:
Let me know how you feel about excluding `/...`. It seemed reasonable at the time but since Ginkgo does nothing to prevent running `ginkgo -r /` I suppose there's no reason to enforce that here either.